tests/nodeos_forked_chain_test.py

```diff
@@ -348,8 +348,12 @@ def getMinHeadAndLib(prodNodes):
         if blockProducer0!=blockProducer1:
             Utils.errorExit("Groups reported different block producers for block number %d. %s != %s." % (blockNum,blockProducer0,blockProducer1))
 
+    #verify that the non producing node is not alive (and populate the producer nodes with current getInfo data to report if
+    #an error occurs)
+    if nonProdNode.verifyAlive():
+        Utils.errorExit("Expected the non-producing node to have shutdown.")
 
-    # ***   Analyze the producers leading up to the block after killing the non-producing node   ***
+    Print("Analyze the producers leading up to the block after killing the non-producing node")
```

Comment:
'Analyzing' rather than 'Analyze'

Reply:
Many changed, but this one and the next didn't.

Reply:
Fixed now.